internal/servers/controller/handlers/errors.go

```diff
@@ -182,7 +183,13 @@ func ErrorHandler(logger hclog.Logger) runtime.ErrorHandlerFunc {
 				errId = "failed_to_generate_error_id"
 			}
 			logger.Error("internal error returned", "error id", errId, "error", inErr)
-			apiErr = getInternalError(errId)
+			var msg string
+			if dErr := errors.Convert(inErr, false); dErr != nil {
+				// This is a domain error, get message to return on API
+				// Do not return the full dErr.Error() as it may contain un-scrubbed info
+				msg = dErr.Msg
```

Comment:
Note to reviewers: are we ok exposing this message to end-users as it is not necessarily as well-formatted as the other messages we return https://github.com/hashicorp/boundary/blob/main/internal/servers/controller/handlers/errors.go#L21